src/tests/ftest/util/test_utils_container.py

```diff
@@ -9,13 +9,109 @@
 from logging import getLogger
 from time import time
 
-from avocado import fail_on
+from avocado import TestFail, fail_on
 from command_utils_base import BasicParameter
 from exception_utils import CommandFailure
 from general_utils import DaosTestError, get_random_bytes
 from pydaos.raw import DaosApiError, DaosContainer, DaosInputParams, str_to_c_uuid
 from test_utils_base import TestDaosApiBase
 
+CONT_NAMESPACE = "/run/container/*"
+
+
+def add_container(test, pool, namespace=CONT_NAMESPACE, create=True, daos=None, **params):
+    """Add a new TestContainer object to the test.
+
+    Args:
+        test (Test): the test to which the container will be added
+        pool (TestPool): the pool to which the container will be added
+        namespace (str, optional): namespace for TestContainer parameters in the test yaml file.
+            Defaults to CONT_NAMESPACE.
+        create (bool, optional): should the container be created. Defaults to True.
+        daos (DaosCommand, optional): daos command object used to create the container.
+            Defaults to calling test.get_daos_command().
+
+    Returns:
+        TestContainer: the new container object
+    """
+    if not daos:
+        daos = test.get_daos_command()
+    container = TestContainer(
+        namespace=namespace, pool=pool, daos_command=daos, label_generator=test.label_generator)
+    container.get_params(test)
+    if params:
+        container.update_params(**params)
+    if create:
+        container.create()
+    test.register_cleanup(remove_container, test=test, container=container)
+    return container
+
+
+def remove_container(test, container):
+    """Remove the requested pool from the test.
+
+    Args:
+        test (Test): the test from which to destroy the container
+        container (TestContainer): the container to destroy
+
+    Returns:
+        list: a list of any errors detected when removing the container
+    """
+    error_list = []
+    test.test_log.info("Destroying container %s", container.identifier)
```

Comment:
`str(TestContainer)` will show the label and UUID
```suggestion
    test.test_log.info("Destroying container %s", str(container))
```

Reply:
Done